wodles/aws/services/inspector.py

```diff
@@ -12,8 +12,15 @@
 sys.path.insert(0, path.dirname(path.dirname(path.abspath(__file__))))
 import aws_tools
 
+INSPECTOR_V1_REGIONS = (
+    'ap-northeast-1', 'ap-northeast-2', 'ap-south-1', 'ap-southeast-2', 'eu-central-1', 'eu-north-1', 'eu-west-1',
+    'eu-west-2', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'
+)
 
-SUPPORTED_REGIONS = (
+INSPECTOR_V2_REGIONS = (
+    'af-south-1', 'ap-east-1', 'ap-northeast-3', 'ap-southeast-1', 'ap-southeast-3', 'ap-southeast-4',
+    'ap-southeast-5', 'ap-southeast-7', 'ap-south-2', 'ca-central-1', 'ca-west-1', 'eu-west-3', 'eu-central-2',
+    'eu-south-1', 'eu-south-2', 'il-central-1', 'me-central-1', 'sa-east-1', 'mx-central-1',
```

Comment:
Given that v2 contains all the regions supported by v1, we could just use them here to avoid repetition

```suggestion
INSPECTOR_V2_REGIONS = INSPECTOR_V1_REGIONS + (
    'af-south-1', 'ap-east-1', 'ap-northeast-3', 'ap-southeast-1', 'ap-southeast-3', 'ap-southeast-4',
    'ap-southeast-5', 'ap-southeast-7', 'ap-south-2', 'ca-central-1', 'ca-west-1', 'eu-west-3', 'eu-central-2',
    'eu-south-1', 'eu-south-2', 'il-central-1', 'me-central-1', 'sa-east-1', 'mx-central-1'
)
```